srtcore/api.cpp

```diff
@@ -1224,6 +1224,12 @@ int srt::CUDTUnited::connect(SRTSOCKET u, const sockaddr* srcname, const sockadd
 
 int srt::CUDTUnited::connect(const SRTSOCKET u, const sockaddr* name, int namelen, int32_t forced_isn)
 {
+    if (!name || size_t(namelen) < sizeof(sockaddr_in))
```

Comment:
Use rather `namelen < int(sizeof(sockaddr_in))`. As this is `int`, we are still interested if this value isn't negative. Comparison against this size of is correct anyway because this is the minimum required in every case - rest of the verifications (whether the size is correct for the given `sa_family`) will be done by `sockaddr_any` below.

Reply:
Ok, I replace `size_t(namelen) < sizeof(sockaddr_in)` for the 2 connect API.